zebra/zebra_fpm_netlink.c

```diff
@@ -327,7 +389,14 @@ static int netlink_route_info_encode(netlink_route_info_t *ri, char *in_buf,
 	req->n.nlmsg_flags = NLM_F_CREATE | NLM_F_REQUEST;
 	req->n.nlmsg_type = ri->nlmsg_type;
 	req->r.rtm_family = ri->af;
-	req->r.rtm_table = ri->rtm_table;
+
```

Comment:
This has nothing to do with rmac updates.  Please add to it's own commit with a proper description of what is going on.

Reply:
Will create another pull request for this.